src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java

```diff
@@ -222,12 +249,244 @@ void method() throws IllegalStateException {
                   }
               }
               """
           )
         );
     }
-}
+
+    @Test
+    void removeEmptyLinesWhenParam() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              class Test {
+                  /**
+                  * Test not removing
+                  *
+                  * Successive newlines
+                  *
+                  *
+                  * @param width
+                  *
+                  *@param height
+                  *
+                  *
+                  */
+                  void method() throws IllegalStateException {
+                  }
+              }
+              """,
+            """
+              class Test {
+                  /**
+                  * Test not removing
+                  *
+                  * Successive newlines
+                  *
+                  *
+                  */
+                  void method() throws IllegalStateException {
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void removeEmptyLinesWhenThrows() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              class Test {
+                  /**
+                  * Test not removing
+                  *
+                  * successive newlines
+                  *
+                  *
+                  * @throws
+                  *
+                  *
+                  */
+                  void method() throws IllegalStateException {
+                  }
+              }
+              """,
+            """
+              class Test {
+                  /**
+                  * Test not removing
+                  *
+                  * successive newlines
+                  *
+                  *
+                  */
+                  void method() throws IllegalStateException {
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void removeEmptyLinesWhenReturn() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              class Test {
+                  /**
+                  * Test not removing
+                  *
+                  * successive newlines
+                  *
+                  *
+                  * @return
+                  *
+                  *
+                  */
+                  void method() throws IllegalStateException {
+                  }
+              }
+              """,
+            """
+              class Test {
+                  /**
+                  * Test not removing
+                  *
+                  * successive newlines
+                  *
+                  *
+                  */
+                  void method() throws IllegalStateException {
+                  }
+              }
+              """
+          )
+        );
+    }
+
+    @Test
+    void issueTest() {
```

Comment:
Perhaps good to reference the issue here, such that we have a link to the context of what prompted this test when reading the code.
```suggestion
    @Issue("https://github.com/openrewrite/rewrite-static-analysis/issues/98")
    void issueTest() {
```